app/src/main/java/com/example/triptracker/model/repository/ItineraryRepository.kt

```diff
@@ -59,7 +59,7 @@ open class ItineraryRepository {
   open fun getPinNames(itinerary: Itinerary): List<String> {
     val pinNames = mutableListOf<String>()
     for (pin in itinerary.pinnedPlaces) {
-      if (pin is Pin) {
+      if (true) {
```

Comment:
Is it the expected behaviour to never enter the else part ? Maybe add a small comment to explain it